zebra/zebra_fpm.c

```diff
@@ -1006,6 +1005,8 @@ static int zfpm_build_route_updates(void)
 					zfpm_g->stats.route_adds++;
 				else
 					zfpm_g->stats.route_dels++;
+			} else {
+				zlog_debug("%s: data_len is 0.", __func__);
```

Comment:
You need to guard this debug statement with a conditional check. Alternatively since we're replacing an assert here, this could probably be a `zlog_error`.

Reply:
`zlog_error` may be OK. But why guard this `zlog_debug` statement?

Reply:
By definition, debugging logs print extraneous information that isn't useful to end users, so as a rule we don't print them by default. To get debugs the user has to enable them.

If this is truly an error condition just change the log function to `zlog_error`.

Reply:
@batmancn if you just change the `debug` to `error` we'll get this in. If you need assistance getting the change in let us know.